README.md

```diff
@@ -210,3 +219,6 @@ un-listed error, simply execute `run.py` again with the same exact options.
 #### Cloudformation
 On occasion the Cloudformation stack will fail to delete properly. Simply go
 into the AWS console and find the Cloudformation stack and delete it.
+
+#### When all hope is lost
+Most likely it's because you have existing settings in your AWS region.  Pick another AWS region and start over. 
```

Comment:
What issues did you have? Did you file issues? How are we supposed to "get it to work" if we don't know what broke? The statement includes an assumption ("you have existing settings") but doesn't explain how to rectify them. That's not really a troubleshooting maneuver.

What if the settings were the same in *every* region? THis troubleshooting step wouldn't work.

Sometimes AWS is broken. I've seen it happen.